server/src/infra/repositories/media.repository.ts

```diff
@@ -23,10 +25,25 @@ export class MediaRepository implements IMediaRepository {
   }
 
   async resize(input: string | Buffer, output: string, options: ResizeOptions): Promise<void> {
-    await sharp(input, { failOn: 'none' })
+    let colorProfile = options.colorspace;
+    if (options.colorspace !== Colorspace.SRGB) {
+      try {
+        const { space } = await sharp(input).metadata();
+        // if the image is already in srgb, keep it that way
+        if (space === Colorspace.SRGB) {
```

Comment:
This is a little strange that we are comparing our internal enum value against the case sensitive value returned from ffprobe. It might be better to use a string instead of the enum. I don't think how we represent the enum needs to be couples with ffprobes value.

Reply:
Yeah, this was a bit awkward since it's actually comparing against another enum that happens to have the same value.